discovery-provider/src/tasks/generate_trending.py

```diff
@@ -141,13 +169,27 @@ def generate_trending(db, time, genre, limit, offset):
             else:
                 track_entry[response_name_constants.repost_count] = 0
 
+            # Populate repost counts with respect to time
+            if track_entry[response_name_constants.track_id] in track_repost_counts_with_time:
+                track_entry[response_name_constants.windowed_repost_count] = \
+                    track_repost_counts_with_time[track_entry[response_name_constants.track_id]]
+            else:
+                track_entry[response_name_constants.windowed_repost_count] = 0
+
             # Populate save counts
             if track_entry[response_name_constants.track_id] in track_save_counts:
                 track_entry[response_name_constants.save_count] = \
                         track_save_counts[track_entry[response_name_constants.track_id]]
             else:
                 track_entry[response_name_constants.save_count] = 0
 
+            # Populate save counts with respect to time
+            if track_entry[response_name_constants.track_id] in track_save_counts_with_time:
+                track_entry[response_name_constants.windowed_save_count] = \
```

Comment:
just confirming - this is bc we utilize both the save_count and windowed_save_count client side right?

Reply:
@raymondjacobson

Reply:
to my understanding, yup!

Reply:
@raymondjacobson

Reply:
clarified offline - this isn't used yet, but we have plans to. resolving